temp/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
I can't. My point is exactly in keeping the .gitignore in temp / log directory. What is the advantage of it? Programmer does not have to create temp, log dirs each time cloning the repo. :)

How can you remove the temp dir when it has to exist in nette app?